### PR TITLE
1.3.3 changelog

### PR DESCRIPTION
**Pull request by yuhan**
## Summary & Motivation
https://github.com/dagster-io/dagster/blob/yuhan/05-03-1.3.3_changelog/CHANGES.md

### CODE REVIEW

**Comment by yuhan** [comment]
Current dependencies on/for this PR:
* master
  * **PR #14071** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/14071" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/14071?utm_source=stack-comment).

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,66 @@
...
+- [dagster-aws] Allow the S3 compute log manager to specify a `show_url_only: true` config option, which will display a URL to the S3 file in dagit, instead of the contents of the log file.
+- [dagster-aws] `PickledObjectS3IOManager` now fully supports loading partitioned inputs.
+- [dagster-azure] `PickedObjectADLS2IOManager` now fully supports loading partitioned inputs.
+- [dagster-gcp] New `GCSResource` and `ConfigurablePickledObjectGCSIOManager` follow the Pythonic resource system. The existing `gcs_resource` and `gcs_pickle_io_manager`  remain supported.
+- [dagster-gcp] New `BigQueryResource` follows the Pythonic resource system. The existing `bigquery_resource`  remains supported.
+- [dagster-gcp] `PickledObjectGCSIOManager` now fully supports loading partitioned inputs.
+- [dagster-postgres] The event watching implementation has been moved from listen/notify based to the polling watcher used by MySQL and SQLite.
+- [dagster-slack] Add `monitor_all_repositories` to `make_slack_on_run_failure_sensor`.
+- [dagster-snowflake] New `SnowflakeResource` follows the Pythonic resource system. The existing `snowflake_resource`  remains supported.
```
Comment (benpankow) [inline CHANGES.md:36]:
```suggestion
- [dagster-snowflake] New `SnowflakeResource` follows the Pythonic resource system. The existing `snowflake_resource`  remains supported.
- [dagster-airbyte] Introduced experimental support for Airbyte Cloud. See the [using Dagster with Airbyte Cloud docs for more information](https://docs.dagster.io/master/integrations/airbyte-cloud).
```

Comment (yuhan) [reply CHANGES.md:36]: should this be in experimental?

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,66 @@
...
+- Fixed an issue with directly invoking an op or asset with a Pythonic config object with a discriminated union did not work properly.
+- Fixes a bug where sensors attached to jobs that rely on resources from Definitions were not provided with the required resource definition.
+
+### Dagster Cloud
+
+- `volumes` and `volumeMounts` values have been added to the agent helm chart.
+
+### Experimental
+
+- [dagster-airbyte] `load_assets_from_airbyte_instance` and `load_assets_from_airbyte_project` now take a `connection_to_auto_materialize_policy_fn` for setting `AutoMaterializePolicy`s on Airbyte assets
```
Comment (benpankow) [inline CHANGES.md:54]:
```suggestion
- [dagster-airbyte] `load_assets_from_airbyte_instance` and `load_assets_from_airbyte_project` now take a `connection_to_auto_materialize_policy_fn` for setting `AutoMaterializePolicy`s on Airbyte assets
- [dagster-airbyte] Introduced experimental support for Airbyte Cloud. See the [using Dagster with Airbyte Cloud docs for more information](https://docs.dagster.io/integrations/airbyte-cloud).
```

File: CHANGES.md (hunk 1 above)
Comment (benpankow) [reply CHANGES.md:36]: 👍 added new comment

File: CHANGES.md [hunk 3]
```diff
@@ -1,5 +1,67 @@
...
+
+        def setup_for_execution(self, context):
+            self._internal_client = MyAPIClient(self.api_key)
+
+        def get_all_items(self):
+            return self._internal_client.items.get()
+    ```
+
+- Added support for specifying input and output config on `ConfigurableIOManager`.
+- `QueuedRunCoordinator` and `SubmitRunContext` are now exposed as pubilc dagster exports.
```
Comment (clairelin135) [inline CHANGES.md:25]: nit: public

**Review by clairelin135 — APPROVED** [review]

File: CHANGES.md (hunk 3 above)
Comment (yuhan) [reply CHANGES.md:25]: 🙏

**Pull request merged**